doc/sphinx/source/users-guide/base/forloops.rst

```diff
@@ -0,0 +1,113 @@
+:title: Users Guide: For loops
+
+.. _ug-forloops:
+
+For-loops
+=========
+
+*For-loops* are Chapel's means of expressing serial iteration as
```

Comment:
That's the slippery slope I'm worried about...  How many languages to compare against?  (and how much detail to go into?)  I'd rather say what we do as clearly as possible here, and save such inter-language discussions for a "Chapel for C programmers" document as much as possible.